watertap/unit_models/coag_floc_model.py

```diff
@@ -369,6 +372,109 @@ def build(self):
             doc="Moles of the produced salts from 1 mole of chemical additives",
         )
 
+        # Below set of Vars are for the power usage of the unit
```

Comment:
I feel `domain` repeats `bounds` in defining `Var` in this section so recommend deleting `domain = ` where bounds are defined.  Correct me if I am wrong.

Reply:
The `domain` is slightly different than `bounds`, but in many cases they have some overlap. The `domain` refers more to the 'type' of numbers that apply for the variable (e.g., complex, real, integers, etc), whereas the `bounds` actually apply inequality constraints to the variables to bound them between the given values. 

In certain cases, such as `domain = NonNegativeReals` with `bounds = (0, None)`, they would effectively have the same impact since the full set of 'NonNegativeReals' is within the specified bounds. 

In other cases, however, it is necessary to specify both (e.g., `domain = PositiveIntegers` with `bounds = (1, 10)` would be needed for integer optimization that is bounded. If only the bounds were provided, then pyomo would assume that non-integers are allowed since the default `domain` is just 'Reals'). 

It is never harmful to apply both a domain and bounds for variables (unless they are contradictory). They each mean something different, even if they sometimes overlap.

Reply:
That is debatable, but there is not really any problem with having both a `domain` and `bounds` in most cases (there are a few edge cases, but they normally don't show up). One reason for having both is so that the `domain` forces an absolute bound that cannot be violated (even if the user tries), whilst the `bounds` can be used to provide adjustable (tighter) limits as needed.